src/styles/block/main.scss

```diff
@@ -0,0 +1,62 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  margin: 0;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+  display: flex;
+}
+
+.calendar {
+  flex-wrap: wrap;
+  gap: $gap;
+  width: 726px;
+  padding: 10px;
+  display: flex;
+}
+
+$days-of-week: (mon, tue, wed, thu, fri, sat, sun);
+
+@for $i from 1 through length($days-of-week) {
+  $day: nth($days-of-week, $i);
+  .calendar--start-day-#{$day} .calendar__day:first-child {
+    margin-left: #{($i - 1) * 101}px;
+  }
+}
+
+.calendar--month-length-28 .calendar__day:nth-child(n + 29),
+.calendar--month-length-29 .calendar__day:nth-child(n + 30),
+.calendar--month-length-30 .calendar__day:nth-child(n + 31),
+.calendar--month-length-31 .calendar__day:nth-child(n + 32) {
```

Comment:
<img width="901" alt="Screenshot 2024-06-06 at 17 25 55" src="https://github.com/mate-academy/layout_calendar/assets/135232001/d90805a9-6ca0-4eeb-b1c1-25a3f11b6b55">

Reply:
Not fixed